docs/Zigbee.md

```diff
@@ -1,4 +1,4 @@
-!!! info "Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi"
+!!! info Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi
```

Comment:
You must leave the quotation mark otherwise the text is not shown anymore

Reply:
Provided new commit. Can you point me out to some documentation how to use the info tag? I tried to browse the md instructions, but none of them mentions such tag :(

Reply:
Hi @sanchosk
There are a few links suggested from the [README.md](https://github.com/tasmota/docs/blob/development/README.md) in the Docs repo.
This one is specifically about [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions) as it is called.
But learned many things the hard-way, making changes and locally regenerating the pages to check the look and feel and validate the syntax. For anything more than just changing a few words/sentence, testing your change locally is a must-do.

Reply:
Thanks for the tip.
I was using local viewer called retext, but it did not understand the tags at all :(
If you look at the history of the commits, I removed them from the whole document and re-formatted it a bit, but those changes were rejected to keep it in line with the original layout.